src/APIM_ARMTemplate/apimtemplate/Extractor/EntityExtractors/TagExtractor.cs

```diff
@@ -8,27 +8,59 @@
 
 namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extract
 {
-    public class TagExtractor: EntityExtractor
+    public class TagExtractor : EntityExtractor
     {
-        public async Task<string> GetTagsAsync(string ApiManagementName, string ResourceGroupName)
+        
+        /// <summary>
+        /// Creates the Tags request URL based on the Azure subscription ID, Resource Group name, and APIM instance name.
+        /// </summary>
+        /// <param name="ApiManagementName"></param>
+        /// <param name="ResourceGroupName"></param>
+        /// <returns>A string representing a request URL</returns>
+        /// <remarks>
+        /// This was split out from the GetTagsAsync method in order to make repeated calls with "nextLink" URLs.
+        /// </remarks>
```

Comment:
Keep fewer comments to keep consistency

Reply:
I wasn't sure about the best way to convey the intent of my changes.  What is the best way to do that?

Reply:
Hi, Kevin, I got your intent for this PR, the logic is good, we may just make some changes before merging it. Here are some tips I think is important:
1. Keep comment with one sentence
2. Keep all the rest call logic like "nextlink" etc within "getTagsAsync" method, and return all tags at one time 
3. A reference could be [Fix 100 API/Ops limit](https://github.com/Azure/azure-api-management-devops-resource-kit/commit/a2fd25e7072f01af138bf415eae0f00b7e5fffef)

Reply:
I'll trim down the comments and take a look at points 2 & 3.  :)